ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -368,11 +377,10 @@ forkBlockProduction maxBlockBodySize IS{..} BlockProduction{..} =
           -- Check whether we adopted our block
           curTip <- atomically $ ChainDB.getTipPoint chainDB
           if curTip == blockPoint newBlock then do
-            -- We measure the time directly.
-            -- We could also measure it when collecting the trace events, but this
-            -- is much more precise.
-            time <- getMonotonicTime
-            trace $ TraceAdoptedBlock currentSlot newBlock txs time
+            -- There is no need to measure the time directly, the precision of
+            -- adding the actual time measurement on the contravariant tracer is
+            -- the same.
```

Comment:
I appreciate the comment, but outside the context of this PR, it seems kind of strange to talk about something that we *used* to do. I think a comment in the PR itself would have made more sense. So I'd remove it.